tests/test-transmuxer.c

```diff
@@ -108,21 +115,98 @@ test_corrupted_empty (void)
   g_unlink (output_file);
 }
 
+static gboolean
+_generate_truncated_file (gchar * src, gchar * dst)
```

Comment:
To make this really simple, have a look at `g_file_get_contents ()` and `g_file_set_contents ()`.

Reply:
Yes, I thought about it, but `g_file_get_contents` reads the whole file while I was planning to read just half of it. 

As the file we are using is small it shouldn't be and issue, so I will change it to make it cleaner.